tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
   "compilerOptions": {
+    "resolveJsonModule": true,
```

Comment:
@mattapperson Following up on https://github.com/elastic/kibana/pull/24425 I added support for importing json files but disabled it for the browser (see `tsconfig.browser.json`).

Reply:
🎉